src/classes/services/ContactService.cls

```diff
@@ -0,0 +1,48 @@
+/**
+ * Created by Mariya on 14.11.2022.
+ */
+
+public with sharing class ContactService {
+    public static void createTeamMember(List<Contact> newContacts){
+
+        List<Contact> contactList = new List<Contact>();
```

Comment:
MINOR
Better name is `contacts`, same for other variables, refer to Code Convention.

Reply:
ok